ophyd/status.py

```diff
@@ -691,9 +691,12 @@ class DeviceStatus(StatusBase):
         status has completed to running callbacks. Default is 0.
     """
 
-    def __init__(self, device, **kwargs):
+    def __init__(self, device, call_stop_on_failure: bool = True, **kwargs):
```

Comment:
I don't think we have type annotations anywhere else in ophyd(-sync), I am not against adding them, but I think adding them one-at-a-time is not a good practice.

Reply:
I'm worried that if we don't add them one-at-a-time, we'll never get around to adding them at all.